pkg/api/types.go

```diff
@@ -1140,6 +1141,22 @@ func (p *Properties) GetKubeProxyFeatureGates() string {
 	return "{}"
 }
 
+// GetKubeProxyFeatureGatesWindowsArguments returns the feature gates string for the kube-proxy arguments in Windows nodes
+func (p *Properties) GetKubeProxyFeatureGatesWindowsArguments() string {
+	featureGates := ""
+	if p.FeatureFlags.IsFeatureEnabled("EnableIPv6DualStack") {
+		featureGates += "'IPv6DualStack=true'"
+	}
+	if p.FeatureFlags.IsFeatureEnabled("EnableWinDSR") {
+		if featureGates != "" {
+			featureGates += ","
+		}
+		// WinOverlay must be set to false
+		featureGates += "'WinDSR=true','WinOverlay=false'"
```

Comment:
Is WinOverlay something users can set in the apimodel?
If so we should add a check in validation.go and warn/fail if both WinOverlay and WinDSR are set and they are mutally exclusive.

Reply:
Doesn't look like we had the ability to add features gates to the Windows kube-proxy prior to this.

Reply:
Agree with @jsturtevant, there is no other way to set feature gates to the Windows kube-proxy.